Assets/Scripts/Tank/TankModel.cs

```diff
@@ -17,10 +17,13 @@ public TankModel(TankType type,float speed, float health, int spawnspot)
         Speed = speed;
         Health = health;
         Spawnspot = spawnspot;
+
+        PlayerId = Random.Range(1, 10000);
```

Comment:
player id should be assigned in the service while creating a controller and should be unique